src/log_to_disk.c

```diff
@@ -106,10 +106,10 @@ void log_to_disk(int from_lan) {
 
     } else {			/* qso from lan */
 
-	/* LOGENTRY contains 82 characters (node,command and logline */
-	g_strlcpy(lan_logline, lan_message + 2, 81);
-	char *fill = g_strnfill(80 - strlen(lan_logline), ' ');
-	g_strlcat(lan_logline, fill, 81);    /* fill with spaces if needed */
+	/* LOGENTRY contains 89 characters (node,command and logline */
+	g_strlcpy(lan_logline, lan_message + 2, LOGLINELEN);
+	char *fill = g_strnfill(LOGLINELEN - 1 - strlen(lan_logline), ' ');
+	g_strlcat(lan_logline, fill, LOGLINELEN);    /* fill with spaces if needed */
```

Comment:
I believe that this would cut away the NL added in the lan code. It is correct, as NL is appended anyway in `store_qso()`.

Reply:
As discussed above the newline is already cut off here.